demisto_sdk/commands/init/contribution_converter.py

```diff
@@ -285,16 +284,27 @@ def convert_contribution_to_pack(self, files_to_source_mapping: Dict = None):
                 basename = os.path.basename(pack_subdir)
                 if basename in {SCRIPTS_DIR, INTEGRATIONS_DIR}:
                     self.content_item_to_package_format(
-                        pack_subdir, del_unified=True, source_mapping=files_to_source_mapping
+                        pack_subdir, del_unified=False, source_mapping=files_to_source_mapping
                     )
 
+                elif basename == 'Playbooks':
+                    files = get_child_files(pack_subdir)
+                    for file in files:
+                        file_name = os.path.basename(file)
+                        if file_name.endswith('.yml'):
+                            self.generate_readme_for_converted_pack(file)
+
                 directories = get_child_directories(pack_subdir)
                 for directory in directories:
-                    unify = Unifier(input=directory, dir_name=basename)
-                    unified_files = unify.merge_script_package_to_yml()
-                    for unified_file in unified_files:
-                        self.generate_readme_for_converted_pack(unified_file)
-                        os.remove(unified_file)
+                    files = get_child_files(directory)
+                    for file in files:
+                        file_name = os.path.basename(file)
+                        if file_name.startswith('integration') or file_name.startswith('script'):
+                            unified_file = file
+                            self.generate_readme_for_converted_pack(unified_file)
+                            os.remove(unified_file)
```

Comment:
Don't we only need to do this in the case of `Integrations` and `Scripts`? If that is indeed the case, then this code block should be moved to beneath line 288 to execute after the call to `self.content_item_to_package_format`.